ibm_catalog.json

```diff
@@ -53,9 +54,26 @@
           "iam_permissions": [
             {
               "role_crns": [
+                "crn:v1:bluemix:public:iam::::serviceRole:Manager",
                 "crn:v1:bluemix:public:iam::::role:Editor"
               ],
-              "service_name": "all-account-management-services"
+              "service_name": "kms",
+              "notes": "Required if you are creating a new Key Protect instance or provisioning associated resources like key rings and keys."
```

Comment:
`[Optional]` prefix is missing.
I'll prefer to write - `Required if KMS encryption is enabled and Key protect is used for encryption.`